<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayer</class>
 <widget class="QWidget" name="MusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>399</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>399</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>音乐播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/player.svg</normaloff>:/Resources/player.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListView" name="musicListView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>161</width>
     <height>281</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>281</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>281</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文仿宋</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListView
{
	background-color:rgba(255,255,255,80);
}
</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QWidget" name="buttonWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border:none;
}</string>
   </property>
   <widget class="QToolButton" name="openDirBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>打开文件夹</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Resources/opendir.svg</normaloff>:/Resources/opendir.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>241</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QToolButton" name="prevSongBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>上一首</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/previous.svg</normaloff>:/Resources/previous.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>播放</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/play.svg</normaloff>:/Resources/play.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nextSongBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>下一首</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/next.svg</normaloff>:/Resources/next.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="loopModeBtn">
       <property name="minimumSize">
        <size>
         <width>43</width>
         <height>43</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>43</width>
         <height>43</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/loopall.svg</normaloff>:/Resources/loopall.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="volBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>音量</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border:none;
}
QToolButton:hover
{
	background-color:rgb(160,160,160);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="Resources.qrc">
         <normaloff>:/Resources/volume.svg</normaloff>:/Resources/volume.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>340</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>391</y>
     <width>82</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="duraLabLayout">
    <item>
     <widget class="QLabel" name="playDuraLab">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="separatorLab">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="durationLab">
      <property name="minimumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>31</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="albumView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>201</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>201</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QGraphicsView
{
	border:none;
	background-color:rgba(255, 255, 255, 20)
}
</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>241</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="infoLabLayout">
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="titleLab">
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;歌曲名称 (Title)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	background-color:rgba(255,255,255,80);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="albAndArtiLabLayout">
      <item>
       <widget class="QLabel" name="artistLab">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;参与创作的艺术家 (ContributingArtist)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color:rgba(255,255,255,80);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="albumLab">
        <property name="font">
         <font>
          <family>华文中宋</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;唱片集 (AlbumTitle)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color:rgba(255,255,255,80);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="listLab">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>54</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>歌曲列表</string>
   </property>
  </widget>
  <widget class="QToolButton" name="modeSwitchBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
	border:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/lightmode.svg</normaloff>:/Resources/lightmode.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lyricLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>315</y>
     <width>300</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color:rgba(255,255,255,80);
	font: 700 12pt &quot;幼圆&quot;;
}</string>
   </property>
   <property name="text">
    <string>这是一栏歌词~~~~~~~~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
